### DISCUSSION TITLE
how to build standalone starter and engine package

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#3263] [Q&A] [answered]
Dear experts,

Currently windows version starter and engine executable have to run intel ONEAPI environment, do we have way to compile a standalone executable which could run on other machine without intel ONEAPI environment. 
If you have related experience, please share it with me. Thank you in advance.

Regards,

Bin

**Comment by OlivierWienholtz** [comment]
Hello,
There is no need to install the entire OneAPI packagez to execute OpenRadioss.
In the Radioss Releases, the OpenRadioss runtime libraries are shipped. You can have a look there & check what you need.
There is one Exception for IntelMPI. IntelMPI is a quite complex package & environment. I recomment to install Standalone version of IntelMPI to execute.

Best Regards,
Olivier

**Reply by halfwitbm** [reply, marked as answer]
Hi Olivier,

Thank you very much for the answer. It is helpful to me.

Regards,

Bin
